python/foglamp/plugins/north/empty/empty.py

```diff
@@ -19,7 +19,11 @@
 _MODULE_NAME = "Empty North Plugin"
 
 _DEFAULT_CONFIG = {
-
+    "destination_type": {
+        "description": "Destination: 1-OMF, 2-Elasticsearch, 3-OCS",
```

Comment:
Elasticsearch? are OMF stats are being sent to `Elasticsearch`?

Reply:
Schema definition for foglamp.destinations in init.sql contains:
>type          smallint                    NOT NULL DEFAULT 1,                         -- Enum : 1: OMF, 2: Elasticsearch

Reply:
Yes, but the schema is a table that is applicable to all north side destinations, this is a configuration for a single plugin which is designed to work with a single destination type.